djl-spring-boot-starter-parent/pom.xml

```diff
@@ -34,8 +34,8 @@
     </distributionManagement>
 
     <properties>
-        <djl.version>0.23.0</djl.version>
-        <java.version>11</java.version>
+        <djl.version>0.26.0</djl.version>
+        <java.version>17</java.version>
```

Comment:
We are not using any JDK 17 APIs, JDK 11 is EOL, but we should keep our API level to 11 to maximize compatability.
User can always use JDK 17 in their production.